task/manager.go

```diff
@@ -1233,6 +1237,23 @@ func (r *Task) podPending(pod *core.Pod) {
 		status,
 		pod.Status.ContainerStatuses...)
 	for _, status := range status {
+		state := status.State
+		if state.Waiting != nil {
+			waiting := state.Waiting
+			reason := strings.ToLower(waiting.Reason)
+			if strings.Contains(reason, "invalid") || strings.Contains(reason, "backoff") {
```

Comment:
Would it be better to specifically match `ImagePullBackOff` and `InvalidImageName`, if those are the only cases we intend to catch?

Reply:
Perhaps but I cannot find a _const_ in v1 or core packages and concerned about making the match too narrow should the strings change.  Also, a little broader match may catch other similar issues.